### PR TITLE
data_acquisition: Fixed up some code, converted to a script

### PR DESCRIPTION
**Pull request by aitikgupta**
Script usage:

```
usage: data_aquisition.py [-h] [--download_dir DOWNLOAD_DIR]
                          [--save_dir SAVE_DIR] [--csv_filename CSV_FILENAME]

optional arguments:
  -h, --help            show this help message and exit
  --download_dir DOWNLOAD_DIR
                        Path to save the downloaded files
  --save_dir SAVE_DIR   Path to save the extracted audio files
  --csv_filename CSV_FILENAME
                        Name of the AudioSet csv file
                        (eval_segments.csv/balanced_train_segments.csv)
```

### CODE REVIEW

File: data_aquisition.py [hunk 1]
```diff
@@ -52,111 +54,131 @@
...
+    base_save_path = str(args.save_dir)
+
+    em_files_path = os.path.join(base_save_path, 'emergency')
+    nonem_files_path = os.path.join(base_save_path, 'nonEmergency')
+
+    ##################################
+    os.chdir(base_download_path)
+
+    # mention the name of the .csv file
+    with open('eval_segments.csv') as csvfile:   # Also use other csv audioset files too extract all the data available in these categories
```
Comment (juhitiwari) [inline data_aquisition.py]: Can you make an argument for the .csv file as well? There are two csv files that need to be downloaded eval and balanced. The idea is to take the csv file name from the user and then prepare the data in that csv in the given folders.

**Review by juhitiwari — CHANGES_REQUESTED** [review]

File: data_aquisition.py (hunk 1 above)
Comment (juhitiwari) [reply data_aquisition.py]: Also please update the README for instructions on script usage

Comment (aitikgupta) [reply data_aquisition.py]: I'll make the changes :+1:

**Comment by aitikgupta** [comment]
@juhitiwari Made the changes 👍🏼

**Review by juhitiwari — APPROVED** [review]

**Pull request merged**